package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon --exec node -r dotenv/config ./src/index.js",
-    "build": "node -r dotenv/config ./src/index.js"
+    "build": "node -r dotenv/config ./src/index.js",
+    "lint": "eslint --fix ./"
```

Comment:
This looks great. BTW, don't you think we should only lint the code in `./src` folder so that we do not deal with `node_modules` or so?!

Reply:
We can achieve this by creating `.eslintignore` in the root folder, you can simply duplicate this https://github.com/andela/andela-tsm-compliance-be/blob/staging/.eslintignore

Reply:
> We can achieve this by creating `.eslintignore` in the root folder, you can simply duplicate this https://github.com/andela/andela-tsm-compliance-be/blob/staging/.eslintignore

That link is broken, unfortunately. I am currently looking at this resource as well that autolints before each commit runs:

Reply:
@gildniy  @timolinn  Or should we combine the two, '.eslintignore' as well as the automation script?

Reply:
Sorry about the link, it's a protected repository. I think anyone should be fine, no need to use the two.